src/mock/v2.0/intent-i.ts

```diff
@@ -0,0 +1,14 @@
+import {
+    closeWindowOnCompletion,
+    onFdc3Ready,
+  } from "./mock-functions";
+  import { DesktopAgent } from "fdc3_2_0";
+  declare let fdc3: DesktopAgent;
+  
+  onFdc3Ready().then(async () => {
+    await closeWindowOnCompletion();
+    fdc3.addIntentListener("sharedTestingIntent2", async (context) => {
```

Comment:
This app is supposed to add the wrong type of intent listener.
```suggestion
    fdc3.addIntentListener("MadeUpIntent", async (context) => {
```